project/gen2.scala

```diff
@@ -120,6 +120,12 @@ object gen2 {
         io.transact(xa).unsafePerformIO
       },
 
+      enum("F2WindowCover") {
+        type F2WindowCoverRec = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'tccValue -> String`.T
+        val io = sql"select id, id tag, short_name, long_name, tcc_value from e_f2_window_cover".query[(String, F2WindowCoverRec)].list
+        io.transact(xa).unsafePerformIO
+      },
```

Comment:
Is a `shapeless` an *official* dependency in `ocs3`? Can we assume we could use `shapeless` indiscriminately anywhere in `ocs3`?

I was reluctant to introduce it for the Seqexec because, for now, I could get away well without it, but if it's already included as dependency I can take advantage of it.